index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
+const mutArrMethods = ['reverse', 'sort', 'splice', 'fill', 'copyWithin']
```

Comment:
Making each of these collections of method names a space-delimited string would save quite a few bytes I think. You can use the `String#includes` method as is done currently.